bin/www

```diff
@@ -1,7 +1,10 @@
 const app = require('../app');
 const debug = require('debug')('surfboard:server');
+const models = require('../models');
```

Comment:
I've seen packages where **every** directory is under /server, with only package.json in the root (or alternatively, they're split between /server and /client). Is this how you're thinking I should refactor the structure?